validation_test.go

```diff
@@ -119,12 +119,11 @@ func TestValidateDatabaseSpanContext(t *testing.T) {
 
 func TestValidateDestinationSpanContext(t *testing.T) {
 	overlong := strings.Repeat("x", 1025)
-	for _, name := range []string{"", overlong} {
+	for range []string{"", overlong} {
```

Comment:
I think we should revert the changes to this file, given that we'll need to keep the API stable